webui/react/src/components/ExperimentCreateModal.tsx

```diff
@@ -337,10 +337,10 @@ const ExperimentCreateModalComponent = ({
     };
     setModalState((prev) => {
       const newModalState = {
+        ...prev,
         config: publicConfig,
-        configString: yaml.dump(publicConfig),
         experiment,
-        isAdvancedMode: false,
+        isAdvancedMode: prev.isAdvancedMode,
```

Comment:
nit: i think this line can be removed due to `...prev`. or do you explicitly declare `isAdvancedMode` here?